syntax/pagesetting.php

```diff
@@ -51,6 +51,10 @@ public function getPType() {
      */
     public function connectTo($mode) {
         $this->Lexer->addSpecialPattern('~~PDF:(?:LANDSCAPE|PORTRAIT)~~', $mode, 'plugin_dw2pdf_pagesetting');
+
+        // this I put here, because it doesnt have sense to create whole new class just for this line
+        // but if things change ...
+		$this->Lexer->addSpecialPattern('<pdfvars[^>]*>[^<]*</pdfvars>' , $mode, 'plugin_dw2pdf_pagesetting');
```

Comment:
Can you do a proposal for a syntax format that matches with the `~~PDF: ...... ~~`? It is fine for me if it is multilined.